app/assets/stylesheets/replies.scss

```diff
@@ -288,8 +288,6 @@ div.post-subheader { width: 100%; }
     label { min-width: 95px; }
   }
 
-  .clear { height: 1px; }
```

Comment:
Moving this because it was broken; this is no longer the correct parent element.